mantidproject/mantid
# EngDiff GSASII tab: Correct tab order and interface height

**Description of work.**

The tab order for the GSASII tab has been defined in the .ui file.
Similarly the tab order has been corrected for the other tabs, resolving a small loop on the focus tab
The GSASII tab used to have quite a large height. I have aligned with the fitting tab to add a scrollbar which appears when the interface height is dragged smaller than the default.

**To test:**

- Select the first widget in a specific tab, e.g. on GSASII the Instrument Group file finder 
- Use Tab and check the focus moves through the widgets in a sensible order (note the overall EngDiff widgets like `RB number`, `?` and `Close` should appear after all widgets in the specific tab.)
 
Fixes #34331 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
